### DISCUSSION TITLE
Can't packer windows 2022 Server

### DISCUSSION THREAD

**Discussion by halinwu** [vmware/packer-examples-for-vsphere#857] [Help] [answered]
I encountered a wierd problem, the update scripte couldn't finished, it  runs repeately .   I don't know what happen , is it problem or I need adjust the update schedule ? 
<img width="975" alt="截屏2024-03-17 00 28 23" src="https://github.com/vmware-samples/packer-examples-for-vsphere/assets/34537132/f0e683b4-fdb6-4e2b-8171-81ec90b30e80">

**Comment by tenthirtyam** [comment, marked as answer]
I've seen issues with one of the January 2024 updates that I addressed in the develop branch along with the transition to Ansible instead of the Windows Update provisioner.

**Reply by tenthirtyam** [reply]
The Microsoft KB was 5034439.

**Reply by halinwu** [reply]
May I skip the update patch ?
